### PR TITLE
Update my.test

### PR DESCRIPTION
**Pull request by GH-integration**

### CODE REVIEW

**Comment by iluxa-com** [comment]
asdf

File: my.test [hunk 1]
```diff
@@ -5,3 +5,6 @@ sadf
 sdfa
 web
 dsfg
+
```
Comment (maxim5) [inline my.test]: 1

Comment (maxim5) [inline my.test]: 2

**Comment by iluxa-com** [comment]
1

**Comment by iluxa-com** [comment]
2

**Comment by iluxa-com** [comment]
3

**Comment by iluxa-com** [comment]
4

**Comment by iluxa-com** [comment]
5

**Comment by iluxa-com** [comment]
6

**Comment by iluxa-com** [comment]
7

**Comment by iluxa-com** [comment]
8

File: my.test (hunk 1 above)
Comment (iluxa-com) [inline my.test:8]: 123

**Pull request closed without merging**